### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by Diegogagan2587**
# Implemented Changes:

- Add posts controler
- add user controller
- add view file for posts
- add a view file for users
- view files only contain placeholder text
- set routes for view files

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class UsersController < ApplicationController
+end
```
Comment (TedLivist) [inline app/controllers/users_controller.rb]: - [ ] Kindly implement the `index` and `show` action. They can be empty for now but they must be implemented.

  ```
  class UsersController < ApplicationController
    def index; end

    def show; end
  end
  ```

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class PostsController < ApplicationController
+end
```
Comment (TedLivist) [inline app/controllers/posts_controller.rb]: - [ ] Kindly implement the `index` and `show` action. They can be empty for now but they must be implemented.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Diegogagan2587,

Good job so far!
✔️ Implementation of nested routes
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 2 above)
Comment (Diegogagan2587) [reply app/controllers/posts_controller.rb]: @TedLivist One more time, thanks for taking the time for this suggestion!

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (Diegogagan2587) [reply app/controllers/users_controller.rb]: @TedLivist Thanks for pointing out! I was not sure about that, I'll doit! thanks!

File: config/routes.rb [hunk 3]
```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
```
Comment (t-yanick) [inline config/routes.rb]: - [ OPTIONAL] 
Great job working on this milestone and implementing the requested changes. Please always avoid keeping commented lines of code in your codebase. It makes it not look professional. You can always use git to revisit old codes.

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED :1st_place_medal: 

Hi @Diegogagan2587 , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Requested changes implemented 👍 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 3 above)
Comment (Diegogagan2587) [reply config/routes.rb]: @t-yanick Thanks for that! I'll remove it!

**Pull request merged**